contentScript.js

```diff
@@ -0,0 +1,94 @@
+let isBlockifyEnabled = false;
+let excludedSites = [];
+let isBlockified = false;
+
+const blockify = () => {
```

Comment:
any chance of for this to be a common import or something like that? becayse this is the same as `blocky-web.user.js`